src/api/c/convolve.cpp

```diff
@@ -198,6 +198,10 @@ af_err af_convolve2(af_array *out, const af_array signal, const af_array filter,
         if (isFreqDomain<2>(signal, filter, domain))
             return af_fft_convolve2(out, signal, filter, mode);
 
+        if (getInfo(signal).dims().ndims()<2 && getInfo(filter).dims().ndims()<2) {
+            return af_convolve1(out, signal, filter, mode, domain);
+        }
+
```

Comment:
Do this before checking for frequency domain and implement it for fftConvolve{3,2} as well